jcl/jpp_configuration.xml

```diff
@@ -37,7 +37,8 @@
 	<!-- START SET DEFINITIONS -->
 	<set
 		label="newflags"
-		flags="OPENJDK_METHODHANDLES"/>
+		flags="AnnotateOverride,Sidecar15,OPENJDK_METHODHANDLES,INLINE-TYPES"/>
+
```

Comment:
This was not merged properly: the list of flags should be:
```
    flags="INLINE-TYPES,OPENJDK_METHODHANDLES"
````

Reply:
Fixed up the rebase issue and added the flag.